lib/pxenforcer.js

```diff
@@ -100,6 +118,7 @@ class PxEnforcer {
         try {
             if (telemetryHandler.isTelemetryCommand(req, this._config)) {
                 this.pxClient.sendEnforcerTelemetry('command', this._config);
+                return cb();
```

Comment:
This is a significant change - have we all aligned on this? I remember this being a discussion and I'm on board -- but we should probably get product to approve it and update the spec before we make this change.

Currently, the spec doesn't say anything about it: https://github.com/PerimeterX/px-enforcer-spec/wiki/Telemetry-Command

Reply:
Removed